### DISCUSSION TITLE
Remix Cache-Control headers seem to be ignored and always respond with Cache MISS

### DISCUSSION THREAD

**Discussion by alp82** [vercel/community#5352] [Feature Question] [answered]
### Question

I have a Remix application and followed the [docs for adding cache headers](https://vercel.com/docs/frameworks/remix#cache-control-headers).

In my pages, I use the following header definition:
```
export function headers() {
  return {
    "Cache-Control": "max-age=300, s-max-age=1800, stale-while-revalidate=7200, stale-if-error=86400",
  };
}
```

But if I refresh the same page multiple times I always get:
```
X-Vercel-Cache: MISS
```

![image](https://github.com/vercel/community/assets/966342/37bde367-6de3-43d4-95c0-4568ac5e3cf3)

Any ideas how to debug or solve this?

**Comment by TooTallNate** [comment, marked as answer]
The correct name is `s-maxage=1800`, instead of `s-max-age=1800` (note the `-` part is removed).

**Reply by alp82** [reply]
thanks a lot for spotting my typo - it works now
